src/main/java/com/project/attendancemanagementsystem/service/ScheduleService.java

```diff
@@ -16,8 +15,9 @@ public ScheduleService(ScheduleRepository scheduleRepository) {
         this.scheduleRepository = scheduleRepository;
     }
 
-    public Schedule findScheduleById(StudentRequest request) {
-        return scheduleRepository.findById(request.getScheduleId())
+    public Schedule findScheduleById(Long scheduleId) {
```

Comment:
Usually, you use `findSmth()` method if you are not sure that returned object exists; For example, methods from a repository are named by this rule. In this case, you either return the object or throw the exception, therefore the returned object always exists, so you can name this method as `getScheduleById();`

Reply:
done